cmd/loadTests.go

```diff
@@ -1666,7 +1673,7 @@ func (h *ConcreteHandlerPipelines) handlePVCS(threadIndex int, framework *framew
 }
 
 type ConcreteHandlerItsPipelines struct {
-	BaseHandler
+    *ConcreteHandlerPipelines // Embedding ConcreteHandlerPipelines
```

Comment:
You should change it to the below to fix the struct embedding
ConcreteHandlerPipelines // Embedding ConcreteHandlerPipelines